destroy-demo.sh

```diff
@@ -19,12 +19,15 @@ DOCKER_SERVER_VERSION_MAJOR=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 1)
 DOCKER_SERVER_VERSION_MINOR=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 2)
 DOCKER_SERVER_VERSION_BUILD=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 3)
```

Comment:
Could you remove these unused variables, `DOCKER_SERVER_VERSION_MAJOR`, `DOCKER_SERVER_VERSION_MINOR`, `DOCKER_SERVER_VERSION_BUILD`...